<launch>
  <!--node pkg="data_processing" name="spacialize_node" type="spacialize.py" output="screen" /-->
  <node pkg="data_processing" name="t265_node" type="t265.py" output="screen" />
  <node pkg="data_processing" name="areamaker_node" type="areaMaker.py" output="screen" />
  <node pkg="data_processing" name="mesh_3D_node" type="animated3Dplot.py" output="screen" />
</launch>
